libfwupd/fwupd-enums.c

```diff
@@ -211,6 +211,8 @@ fwupd_device_flag_to_string(FwupdDeviceFlags device_flag)
 		return "unsigned-payload";
 	if (device_flag == FWUPD_DEVICE_FLAG_EMULATED)
 		return "emulated";
+	if (device_flag == FWUPD_DEVICE_FLAG_ALLOW_EMULATE_SAVE)
+		return "allow-emulate-save";
```

Comment:
@superm1 i'm not sure about the name of this flag. It unfortunately does need to be public (and not internal / private) as we need to filter it client side when removing watched devices.

Reply:
How about `allow-emulation`

Reply:
It's less "allowing emulation" and more "start recording this device"

Reply:
@superm1 Counter proposal, how about `recording-for-emulation` or `emulation-recording` or `emulation-events` or `save-emulation-events` or `emulation-tag`? My favorite there is `emulation-tag` as then we can make the fwupdmgr argument `emulate-tag` and `emulate-untag`.

Reply:
I like `emulation-tag` / untag the most of your suggestions.  However `tag-for-emulation` and `untag-for-emulation` would work too.

Reply:
I've changed the flag to be `emulation-tag`, and the `fwupdmgr` command to be `emulate-tag`

Reply:
Yup, I've used emulation, it's a lot more consistent now.